common4j/src/main/com/microsoft/identity/common/java/logging/DiagnosticContext.java

```diff
@@ -67,6 +68,21 @@ public IRequestContext getRequestContext() {
         return REQUEST_CONTEXT_THREAD_LOCAL.get();
     }
 
+
+    /**
+     * Utility method to return the correlation id for current thread.
+     * @return Correlation id or "UNSET"
+     */
+    public String getThreadCorrelationId() {
+        IRequestContext context = getRequestContext();
+        String threadId = context.get(DiagnosticContext.CORRELATION_ID);
+        if (threadId == null)
+        {
+            threadId = UNSET;
+        }
+        return threadId;
```

Comment:
```suggestion
        final String correlationId = context.get(DiagnosticContext.CORRELATION_ID);
        if (correlationId == null)
        {
           return UNSET;
        }
        return correlationId;
```

Reply:
I have changed the variable name but kept the single return statement. I prefer single return statement in short methods. I also find that the `final` qualifier doesn't add to more readability in short methods.